mRemoteV1/mRemoteV1.csproj

```diff
@@ -59,23 +59,40 @@
     <Reference Include="Geckofx-Winforms, Version=45.0.32.0, Culture=neutral, PublicKeyToken=3209ac31600d1857, processorArchitecture=x86">
       <HintPath>..\packages\Geckofx45.45.0.32\lib\net45\Geckofx-Winforms.dll</HintPath>
     </Reference>
+    <Reference Include="Google.Protobuf, Version=3.5.1.0, Culture=neutral, PublicKeyToken=a7d26565bac4d604, processorArchitecture=MSIL">

```

Comment:
Just eyeballing the diff on GitHub (I didn't do an in depth review, so I apologize if I've missed something) - I don't see any use of this reference. If it's not in use, please remove the reference and the nuget.

Reply:
Done - pushed a rebase to include this in the main commit